client/packages/common/src/hooks/useNativeClient/useNativeClient.ts

```diff
@@ -94,7 +100,7 @@ export const useNativeClient = ({
     const timeoutTimer = setTimeout(() => setTimedOut(true), DISCOVERY_TIMEOUT);
 
     const pollTimer = setInterval(async () => {
-      const servers = await nativeAPI.discoveredServers();
+      const servers = (await nativeAPI.discoveredServers()).servers;
```

Comment:
perhaps neater?
```suggestion
      const { servers } = await nativeAPI.discoveredServers();
```
Though I have to say, having `discoveredServers` return a property of `servers` seems like duplication.
`const { servers } = await nativeAPI.discover();` reads better to me